grid/address/ip-geolocation/maps/ipdata.suma

```diff
@@ -0,0 +1,70 @@
+profile = "address/ip-geolocation@1.0"
+provider = "ipdata"
+
+// IPData documentation: https://docs.ipdata.co/
+map IpGeolocation {
+  ipAddress = ''
+
+  set if (input.ipAddress) {
+    ipAddress = input.ipAddress
+  }
+
+  http GET "/{ipAddress}" {
```

Comment:
You could use a response fields filter to avoid transmitting data we don't expose: https://docs.ipdata.co/api-reference/selecting-response-fields

Reply:
done